cmd/kubeseal/main_test.go

```diff
@@ -377,7 +417,7 @@ func TestMergeInto(t *testing.T) {
 
 func TestVersion(t *testing.T) {
 	var buf strings.Builder
-	err := run(&buf, "", "", "", "", true, false, false, false, false, nil, "")
+	err := run(&buf, "", "", "", "", true, false, false, false, false, nil, "", false, nil)
```

Comment:
ouch :/

Reply:
Yeah, I'd really like to move away from this pattern...